config/locales/ne.yml

```diff
@@ -0,0 +1,1338 @@
+      awaiting_return:
+      backordered:
+      canceled:
+      cart:
+      checkout:
+      closed:
+      complete:
+      completed:
+      confirm:
+      delivery:
+      errored:
+      failed:
+      given_to_customer:
+      invalid:
+      manual_intervention_required:
+      on_hand:
+      open:
+      order:
+      payment:
+      pending:
+      processing:
+      ready:
+      reimbursed:
+      resumed:
+      returned:
+      shipped:
+      void:
+    states: काउन्टीहरू
+    states_required: काउन्टी आवश्यक छ
+    status: स्थिति
+    stock:
+    stock_location: स्टक स्थान
+    stock_location_info: स्टक स्थान जानकारी
+    stock_locations: स्टक स्थानहरू
+    stock_locations_need_a_default_country:
+    stock_management: स्टक प्रबन्धन
+    stock_management_requires_a_stock_location: स्टक प्रबन्ध गर्नका लागि कृपया स्टक स्थान सिर्जना गर्नुहोस्।
+    stock_movements: स्टक चाल
+    stock_movements_for_stock_location: '%{stock_location_name} का लागि स्टक चाल'
+    stock_successfully_transferred: स्टक सफलतापूर्वक स्थानहरू बीच हस्तान्तरण गरियो।
+    stock_transfer: स्टक स्थानान्तरण
+    stock_transfers: स्टक स्थानान्तरण
+    stop: रोक
+    store: स्टोर
+    street_address: सडक ठेगाना
+    street_address_2: सडक ठेगाना (जारी)
+    subtotal: उप कूल
+    subtract: घटाउ
+    success:
+    successfully_created: "%{resource} सफलतापूर्वक सिर्जना गरिएको छ!"
+    successfully_refunded:
+    successfully_removed: "%{resource} सफलतापूर्वक हटाइएको छ!"
+    successfully_signed_up_for_analytics: सफलतापूर्वक Spree विश्लेषक को लागी साइन अप
+    successfully_updated: "%{resource} सफलतापूर्वक अपडेट गरियो!"
+    summary:
+    tax: कर
+    tax_categories: कर कोटिहरू
+    tax_category: कर कोटि
+    tax_code:
+    tax_included:
+    tax_rate_amount_explanation: कर दरहरू गणनामा सहायताका लागि दशमलव रकम हुन्, (अर्थात् यदि कर दर 5% छ भने ०.०5 प्रविष्ट गर्नुहोस्)
+    tax_rates: कर दरहरू
+    taxon: ट्याक्सन
+    taxon_edit: ट्याक्सन सम्पादन गर्नुहोस्
+    taxon_placeholder: एक ट्याक्सन जोड्नुहोस्
+    taxon_rule:
+      choose_taxons:
+      label:
+      match_all:
+      match_any:
+    taxonomies: वर्गीकरण
+    taxonomy: वर्गीकरण
+    taxonomy_edit: वर्गीकरण सम्पादन गर्नुहोस्
+    taxonomy_tree_error: अनुरोध गरिएको परिवर्तन स्वीकार गरिएको छैन र रूख यसको अघिल्लो स्थितिमा फिर्ता आएको छ, कृपया फेरि प्रयास गर्नुहोस्।
+    taxonomy_tree_instruction: "* बच्चा थप्न, मेटाउने वा क्रमबद्ध गर्नका लागि मेनू पहुँच गर्न रूखमा भएको बच्चाको दायाँ क्लिक गर्नुहोस्।"
+    taxons: करहरू
+    test: परीक्षण
+    test_mailer:
+      test_email:
+        greeting: बधाई छ!
+        message: यदि तपाईंले यो ईमेल प्राप्त गर्नुभयो भने, तपाईंको ईमेल सेटिंग्स सही छ।
+        subject: परीक्षण मेल
+    test_mode: परीक्षण मोड
+    thank_you_for_your_order: तपाईको व्यवसायका लागि धन्यबाद। तपाईंको रेकर्डहरूको लागि कृपया यस कन्फर्मेसन पृष्ठको प्रतिलिपि प्रिन्ट गर्नुहोस्।
+    there_are_no_items_for_this_order: यस अर्डरको लागि त्यहाँ कुनै आईटमहरू छैनन्। आदेश जारी गर्न एक वस्तु जोड्नुहोस्।
```

Comment:
यस अर्डरको लागि त्यहाँ(should be yeha) कुनै आईटमहरू छैनन्। 

आदेश जारी गर्न एक वस्तु जोड्नुहोस्।
order garnako lagi kamti ma euta samagri add garnuhos